src/frontend/src/routes/(app)/activity/+layout.svelte

```diff
@@ -0,0 +1,8 @@
+<script lang="ts">
+	import MultipleListeners from '$lib/components/core/MultipleListeners.svelte';
+	import { enabledNetworkTokens } from '$lib/derived/network-tokens.derived';
+</script>
+
+<MultipleListeners tokens={$enabledNetworkTokens}>
```

Comment:
the `AuthGuard` is already in the `+layout` of the parent `(app)` folder; so it should not be mounted anyway if not logged in

Reply:
So, we do not need `TokensSignedOut` anymore? I asked the question because I had a look at current code and noticed that we currently mount, or not, the `MultipleListeners` according `$authSignedIn` within the `Tokens.svelte` component.

Reply:
You are right, we don't need it anymore

Reply:
Thanks for the PR.

Then, second part of my question, should `MultipleListeners` find place in the `+layout` or in the `+page`?

Reply:
So, instead of having, single listeners in the Token list component, we have just one `MultipleListeners` in the main `+layout `, right?

If yes, can I do it in another PR? It seems something that should go by itself

Reply:
I think you did not understand my question. I meant why implementing `MultipleListeners` in `+layout.ts` - i.e. why not implementing it in `+page.svelte`?

It feels something more related to the page than it's layout.

Reply:
ah! now I understand! yes, seems right! changed